packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -25,8 +25,15 @@ export default {
       }
     },
     stats: {
+      // 1 second intervals
       interval: 1000,
-      historyMax: 1800
+      // hold the last 2 minute of a calls data
+      historyMax: 120,
+      // should we move these to a separate config key?
+      // send a metrics event automatically
+      autoMediaMetrics: false,
+      // dispatch data every 10 seconds
+      mediaMetricsInterval: 10
```

Comment:
Why is this different than the other interval? Seems like the naming could be tweaked here, too. Love all the configuration options though.